docs/developer/file-structure.rst

```diff
@@ -285,6 +285,8 @@ levelling. PyMedPhys' automated test suite includes a Python package called
 For a further, in-depth explanation of the philosophy behind levelling
 dependencies, see the :ref:`john-lakos` section.
 
+|
```

Comment:
In the CSS I have now globally formatted greater section breaks by default:

https://github.com/pymedphys/pymedphys/blob/master/docs/_static/theme_overrides.css#L104-L106

I can make these section breaks even larger if you like?